src/app/Controllers/MoviesController.cs

```diff
@@ -86,7 +86,7 @@ public async Task<IActionResult> GetMoviesAsync([FromQuery]string q = "", [FromQ
                 };
             }
 
-            catch (System.AggregateException age)
+            catch (AggregateException age)
```

Comment:
Should make this change in the other controllers/files too (I saw it in ActorsController, FeaturedController, GenresController, and CosmosHealthCheck